_test/shared/matlab_xunit/xunit/runtests.m

```diff
@@ -171,7 +171,7 @@
         else
             ddot_ind = strfind(arg,':');
             if isempty(ddot_ind) || ~ispc ||...
-                    (ispc && (ddot_ind == 2)) % C: or other drive on Windows and full path provided.
+                    (ispc && numel(ddot_ind) == 1 && (ddot_ind(1) == 2)) % C: or other drive on Windows and full path provided.
```

Comment:
this is already a complex `if` statement; it would be clearer to extract the "contains drive letter check" as a helper function so it collapses to:
`if isempty(ddot_ind) || ~ispc || contains_full_windows_path(ddot_ind)`

Reply:
fair enough -- fixed.